theme_orchid/views/snippets/s_image_hexagonal.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_hexagonal" inherit_id="website.s_image_hexagonal">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        A place of paradise for flower lovers
+    </xpath>
+    <!-- Images -->
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_masonry_block_default_image_1/web_editor/composition/composition_square_1.svg</attribute>
```

Comment:
The `data-shape` attribute should also be adapted.

Reply:
All three have been fixed, thanks !